internal/sock/sock_unsupported.go

```diff
@@ -0,0 +1,10 @@
+//go:build plan9 || js
+
+package sock
+
+// plan9/js doesn't declare these constants
+const (
+	SHUT_RD = iota
+	SHUT_RDWR
```

Comment:
Should we make SHUT_RDWR equal SHUT_RD|SHUT_WR?

Probably better if SHUT_RD isn't zero as well otherwise it's difficult to not shut the read end of the connection.

Reply:
https://github.com/tetratelabs/wazero/pull/1614/commits/0b5b6fed86700db715b29280ed8fb700c01ef732

Reply:
good catch!